### ISSUE TITLE
Description ...

### ISSUE THREAD

**Issue by Indribell** [mojura/enkodo#3]
> Enkodo is a no-frills encoder/decoder. It is focused around speed and simplicity.

Looking over the code it seems that the package is being very selective in its encoding. Basic types are supported but more advanced like arrays and maps seem to be missing any support.

This may need to be better explained in the readme section.

**Comment by itsmontoya** [comment]
You're totally correct! Although this is planned for the future, it's not available currently. I'll add a note to the readme :)
